src/pages/index.js

```diff
@@ -76,26 +74,28 @@ const userInfo = new UserInfo(
   ".profile__image"
 );
 
-api.getUserInfo().then((userData) => {
-  console.log(userData);
-  userInfo.setUserInfo(userData.name, userData.about);
-});
+api
+  .getUserInfo()
+  .then((userData) => {
+    console.log(userData);
+    userInfo.setUserInfo(userData.name, userData.about);
+    userInfo.setAvatar(userData.avatar);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 const confirmDeleteModal = new PopupWithConfirmation(
   "#confirm-delete-modal",
-  (id) => {
-    confirmDeleteModal.renderLoading(true, "Deleting...");
-
+  () => {
     api
       .removeCard(id)
       .then(() => {
+        console.log("testing");
         confirmDeleteModal.close();
       })
       .catch((err) => {
         console.error(err);
```

Comment:
* You havent' corrected something https://github.com/sheenasli/se_project_aroundtheus/pull/1#discussion_r1342908896 


You don't need to add this logics here because you don't have `id` for line 92